spec/dishes_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'dishes'
+
+describe Dishes do
+
+
+  it { is_expected.to respond_to(:menu)}
+
+
+  it 'allows user to see a list of dishes with prices' do
+    dish = Dishes.new
+    expect(dish.menu).to include({ "shoes" => 78 })
```

Comment:
...this could have been something like 
' expect(dishes.menu).to contain_exactly({"Seeds" => 50, "Eyeballs" => 25, "Camel Breast" => 40, "shoes" => 78 })'
Which i guess is only a little more comprehensive, would this test pass?